src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  title,
+  description,
```

Comment:
`description` is a not required field, it might be undefined, you should set some default value for it